### ISSUE TITLE
define autocomplete prompts

### ISSUE THREAD

**Issue by unicornbloods** [Strum355/mcshader-lsp#23]
Hey, I was just wondering if ya could do auto-complete prompts for #define that people make.
So like if I did a #define meow and I started typing out meow it would show the prompt to auto complete it

**Comment by Strum355** [comment]
Yes this will hopefully become a thing, for more than just defines, but functions and variables too
